app/views/nsm/additional_fees/show.html.erb

```diff
@@ -23,7 +23,7 @@
   <div class="govuk-grid-column-three-quarters">
 
     <h1 class="govuk-heading-xl">
-        <%= t('.heading', type: item.type) %>
+      <%= t(".heading.#{item.any_adjustments? ? 'true' : 'false'}", type: item.type) %>
```

Comment:
Why can't this just be a simple if/else?

Reply:
following existing pattern of coding, same pattern used in all other places.

Reply:
I meant an if/else in the view

```erb
<% if item.any_adjustments? %>
  <%= t(".adjustments_heading") %>
<% else %>
  <%= t(".heading") %>
<% end %>
```

Or something like that

Reply:
I meant the same, please review show/edit in disbursement,work items